neuralmonkey/dataset.py

```diff
@@ -14,6 +14,7 @@
 
 SERIES_SOURCE = re.compile("s_([^_]*)$")
 SERIES_OUTPUT = re.compile("s_(.*)_out")
+PREPROCESSED_SERIES = re.compile("pre_([^_]*)$")
```

Comment:
Jo, viz `tests/post-edit.ini`. 

`pre_edits=<preprocess>`

udělá, že se zavolá call na `<preprocess>` a z výsledku bude series, co se jmenuje `edits`. Přijde mi to docela v duchu zbytku tvoření toho datasetu.